### ISSUE TITLE
[BUG] In the pong example paddles and ball are not rendered

### ISSUE THREAD

**Issue by shinglyu** [amethyst/amethyst#1650]
## Description

When running the `master` branch example of `pong`, the game is running but the paddles and ball are not rendered correctly.

## Reproduction Steps

Run `cargo run --example pong --features "vulkan"`

## What You Expected to Happen

The demo runs correctly with visible paddles and ball

## What Actually Happened

The game runs, sound and score work fine, but the paddles and ball are not visible.

## Screenshots and Logs

![Selection_138](https://user-images.githubusercontent.com/3250983/58386987-ef559300-8007-11e9-8ebe-97e9f8e74596.png)

## Your Environment

1. Operating System: Ubuntu Linux
2. Operating System Version: 16.04.6 LTS, Linux kernel 4.15.0-50-generic x86_64
3. Version of Amethyst: master branch commit 85365d5543b9be0daa891706c5d46c80ee38f9ca
4. Vulkan version: mesa driver; Vulkan API Version: 1.0.61; 
5. Video card: VGA compatible controller: Intel Corporation HD Graphics 620 (rev 02)
6. Rust version: tried both stable (1.34.0) and nightly

**Comment by shinglyu** [comment]
Please let me know if any specific log or system information dump will be useful. I'm happy to provide them.

**Comment by RybekU** [comment]
Could you confirm if it still happens with this version of initialise_camera?
```
/// Initialise the camera.
fn initialise_camera(world: &mut World) {
    let mut transform = Transform::default();
    transform.set_translation_xyz(0.0, 0.0, 1.0);

    world
        .create_entity()
        // A default camera can be created with standard_2d, but we instead create a camera
        // which is centered on our gameplay area (ARENA)
        .with(Camera::from(Projection::orthographic(
            0.0,
            ARENA_WIDTH,
            0.0,
            -ARENA_HEIGHT,
            0.1,
            2000.0,
        )))
        .with(transform)
        .build();
}
```
If it will work fine with this version it means it has something to do with Projection::ortographic not working like it used to.

**Comment by minecrawler** [comment]
Fixes it for me, though also inverts at least the input

**Comment by RybekU** [comment]
> 
> 
> Fixes it for me, though also inverts at least the input - so I guess the drawing is wrong.

The input being inverted is because of the config file. The code before edit was inverting camera and thus making input work as expected (up arrow making the paddle go up), but also making the sprites appear upside down (not noticeable until i changed the color on one side).

**Comment by shinglyu** [comment]
The code edit fixed it for me as well. So was the camera set in the wrong way or was there a bug for orthographic projection?

**Comment by jojolepro** [comment]
Nothing renders for me.

**Comment by shinglyu** [comment]
@RybekU how would you suggest we move this issue forward? Should I apply your patch and edit the config file to invert the control? Or is there a more fundamental fix for it?

**Comment by Dispersia** [comment]
I did a PR https://github.com/amethyst/amethyst/pull/1673, depending on which way it goes it should be fixed under it

**Comment by torkleyy** [comment]
Can you please try again once #1679 is merged? It should fix the issue.

**Comment by Frizi** [comment]
I'm assuming this is fixed by #1679
